### PR TITLE
7.about page has been created

### PR DESCRIPTION
**Pull request by ertantherock**
In order to see about page, check and approve my pull request please.

### CODE REVIEW

File: SWE574/core/views.py [hunk 1]
```diff
@@ -57,6 +57,16 @@ def book_post(request: object):
 def profile(request, profile_owner_username):
     return profile_page_handler_main(request=request, profile_owner_username=profile_owner_username)
 
+@login_required(login_url="core:signin")
+def about2(request):
```
Comment (dberkerdem) [inline SWE574/core/views.py:61]: delete this function (about2)

File: SWE574/core/views.py [hunk 2]
```diff
@@ -57,6 +57,16 @@ def book_post(request: object):
...
+    return render(request, "about2.html")
+
+@login_required(login_url="core:signin")
+def about(request, profile_owner_username):
```
Comment (dberkerdem) [inline SWE574/core/views.py:65]: try using following

def about(request):
    if request.user.is_authenticated:
        return render(request, "about.html")
    else:
        return render(request, "about2.html")

File: SWE574/core/urls.py [hunk 3]
```diff
@@ -18,6 +18,8 @@
     path("preview/", generate_preview.generate_preview, name="generate"),
     path("search", views.search, name="search"),
     path("follow", views.follow, name="follow"),
+    path('about2/', views.about2, name='about2'),
+    path('profile/<str:profile_owner_username>/about/', views.about, name='about'),
```
Comment (dberkerdem) [inline SWE574/core/urls.py:22]: this line is not required

File: SWE574/core/urls.py [hunk 4]
```diff
@@ -18,6 +18,8 @@
     path("preview/", generate_preview.generate_preview, name="generate"),
     path("search", views.search, name="search"),
     path("follow", views.follow, name="follow"),
+    path('about2/', views.about2, name='about2'),
```
Comment (dberkerdem) [inline SWE574/core/urls.py:21]: change this from about2 to about
```
path("about", views.about, name='about'),
```

File: SWE574/core/templates/about2.html [hunk 5]
```diff
@@ -0,0 +1,137 @@
+{% extends 'base.html' %}
```
Comment (dberkerdem) [inline SWE574/core/templates/about2.html:1]: The only difference between this html and about.html must be the navbar. 
Navbar of this html should stay the same whereas the navbar of about.html must be inherited from navbar.html by using 
```
{% include 'navbar.html' %}
```

File: SWE574/core/templates/about.html [hunk 6]
```diff
@@ -0,0 +1,136 @@
+{% extends 'base.html' %}
```
Comment (dberkerdem) [inline SWE574/core/templates/about.html:1]: See the note in about2.hmtl

File: SWE574/core/templates/empty.html [hunk 7]
```diff
@@ -28,7 +28,7 @@ <h4 class="m-t-10 m-b-5">Sean Ngu</h4>
                   <!-- BEGIN profile-header-tab -->
                   <ul class="profile-header-tab nav nav-tabs">
                      <li class="nav-item"><a href="https://www.bootdey.com/snippets/view/bs4-profile-with-timeline-posts" target="__blank" class="nav-link_">POSTS</a></li>
-                     <li class="nav-item"><a href="https://www.bootdey.com/snippets/view/bs4-profile-about" target="__blank" class="nav-link_">ABOUT</a></li>
+                     <li class="nav-item"><a href="https://www.bootdey.com/snippets/view/bs4-profile-about" target="about2/" class="nav-link_">ABOUT</a></li>
```
Comment (dberkerdem) [inline SWE574/core/templates/empty.html:31]: Is this required?

File: SWE574/core/templates/navbar.html [hunk 8]
```diff
@@ -46,7 +46,7 @@
                        href="{% url 'core:profile' profile_owner_username=request.user.username %}">Profile</a>
                     <a class="dropdown-item text-white" href="/settings">Settings</a>
                     <a class="dropdown-item text-white" href="/settings">{{ request.user.username }}</a>
-                    <a class="dropdown-item text-white" href="#">About</a>
+                    <a class="dropdown-item text-white" href="about2/">About</a>
```
Comment (dberkerdem) [inline SWE574/core/templates/navbar.html:49]: Consider the suggested changes in views and urls. Use only about/

File: SWE574/core/templates/signin.html [hunk 9]
```diff
@@ -13,7 +13,7 @@
         </li>
         <li class="nav-item">
           <!-- TODO: Fill Here -->
-          <a href="" class="nav-link px-3 text-white">About</a>
+          <a href="about2/" class="nav-link px-3 text-white">About</a>
```
Comment (dberkerdem) [inline SWE574/core/templates/signin.html:16]: Change to about/

File: SWE574/core/templates/signup.html [hunk 10]
```diff
@@ -12,7 +12,7 @@
           <a href="" class="nav-link px-3 text-white">Home</a>
         </li>
         <li class="nav-item">
-          <a href="" class="nav-link px-3 text-white">About</a>
+          <a href="about2/" class="nav-link px-3 text-white">About</a>
```
Comment (dberkerdem) [inline SWE574/core/templates/signup.html:15]: Change to about/

**Review by dberkerdem — CHANGES_REQUESTED** [review]

File: SWE574/core/templates/signin.html (hunk 9 above)
Comment (ertantherock) [reply SWE574/core/templates/signin.html:16]: Tamam

File: SWE574/core/views.py (hunk 1 above)
Comment (ertantherock) [reply SWE574/core/views.py:61]: Teşekkürler

File: SWE574/core/views.py (hunk 2 above)
Comment (ertantherock) [reply SWE574/core/views.py:65]: Daha mantıklı teşekkürler

**Comment by ertantherock** [comment]
All changes have been made

**Pull request closed without merging**